frontend/client/app.js

```diff
@@ -12,21 +12,24 @@ import auth from './auth'
 import lodash from 'lodash'
 import VueLodash from 'vue-lodash'
 
-Vue.prototype.$http = axios
-Vue.axios = axios
+const axiosInstance = axios.create({
+  baseURL: './'
```

Comment:
This means that `./` will always be the default. Shouldn't this be just `/`?

Reply:
it will always be relative to the current base and since its a SPA that will be where ever it is served from. `/` would be absolute so we would have to do some trickery of setting the conditional root and I am lazy :)